src/cascadia/TerminalSettingsEditor/Appearances.idl

```diff
@@ -67,8 +69,8 @@ namespace Microsoft.Terminal.Settings.Editor
         IInspectable CurrentAdjustIndistinguishableColors;
         Windows.Foundation.Collections.IObservableVector<Microsoft.Terminal.Settings.Editor.EnumEntry> AdjustIndistinguishableColorsList { get; };
 
-        Microsoft.Terminal.Settings.Model.ColorScheme CurrentColorScheme;
-        Windows.Foundation.Collections.IObservableVector<Microsoft.Terminal.Settings.Model.ColorScheme> ColorSchemeList { get; };
+        ColorSchemeViewModel CurrentColorScheme;
+        Windows.Foundation.Collections.IObservableVector<ColorSchemeViewModel> ColorSchemeList { get; };
```

Comment:
Can we remove `ColorSchemeList` and use `SchemesPageVM.AllColorSchemes()` instead?

Reply:
Yep! Good call, changed